api/namex/constants/__init__.py

```diff
@@ -271,6 +282,9 @@ class LegacyEntityTypes:
     ('CCR', EntityTypes.CORPORATION.value, RequestAction.RESUBMIT.value),
     ('CT', EntityTypes.CORPORATION.value, RequestAction.MVE.value, True),
     ('CT', EntityTypes.CORPORATION.value, RequestAction.RESUBMIT.value),
+    ('CT', EntityTypes.CONTINUE_IN.value, RequestAction.RESUBMIT.value),
+    ('CT', EntityTypes.CONTINUE_IN.value, RequestAction.REH.value),
+    ('CT', EntityTypes.CONTINUE_IN.value, RequestAction.CHG.value),
```

Comment:
No. CT is used for MVE (see line 283).

Here, please add the following for name change NR types:
```
    ('CCR', EntityTypes.CONTINUE_IN.value, RequestAction.CHG.value, True),
    ('CCR', EntityTypes.CONTINUE_IN.value, RequestAction.RESUBMIT.value),

    ('CCC', EntityTypes.CONTINUE_IN_CCC.value, RequestAction.CHG.value, True),
    ('CCC', EntityTypes.CONTINUE_IN_CCC.value, RequestAction.RESUBMIT.value),

    ('CUL', EntityTypes.CONTINUE_IN_CUL.value, RequestAction.CHG.value, True),
    ('CUL', EntityTypes.CONTINUE_IN_CUL.value, RequestAction.RESUBMIT.value, True),

    ('BEC', EntityTypes.CONTINUE_IN_CBEN.value, RequestAction.CHG.value, True),
    ('BEC', EntityTypes.CONTINUE_IN_CBEN.value, RequestAction.RESUBMIT.value),
```

Also add this missing one:
```
    ('CUL', EntityTypes.UNLIMITED_LIABILITY_COMPANY.value, RequestAction.RESUBMIT.value, True),
```

Also add these for restoration NR types:
```
    ('RCR', EntityTypes.CONTINUE_IN.value, RequestAction.REST.value, True),
    ('RCR', EntityTypes.CONTINUE_IN.value, RequestAction.REH.value),
    ('RCR', EntityTypes.CONTINUE_IN.value, RequestAction.REN.value),
    ('RCR', EntityTypes.CONTINUE_IN.value, RequestAction.RESUBMIT.value),

    ('RCC', EntityTypes.CONTINUE_IN_CCC.value, RequestAction.REST.value, True),
    ('RCC', EntityTypes.CONTINUE_IN_CCC.value, RequestAction.REH.value),
    ('RCC', EntityTypes.CONTINUE_IN_CCC.value, RequestAction.REN.value),
    ('RCC', EntityTypes.CONTINUE_IN_CCC.value, RequestAction.RESUBMIT.value),

    ('RUL', EntityTypes.CONTINUE_IN_CUL.value, RequestAction.REST.value, True),
    ('RUL', EntityTypes.CONTINUE_IN_CUL.value, RequestAction.REH.value),
    ('RUL', EntityTypes.CONTINUE_IN_CUL.value, RequestAction.REN.value),
    ('RUL', EntityTypes.CONTINUE_IN_CUL.value, RequestAction.RESUBMIT.value),

    ('BERE', EntityTypes.CONTINUE_IN_CBEN.value, RequestAction.REST.value, True),
    ('BERE', EntityTypes.CONTINUE_IN_CBEN.value, RequestAction.REH.value),
    ('BERE', EntityTypes.CONTINUE_IN_CBEN.value, RequestAction.REN.value),
    ('BERE', EntityTypes.CONTINUE_IN_CBEN.value, RequestAction.RESUBMIT.value),
```

Reply:
Looks good. Resolving.